src/zimdump.cpp

```diff
@@ -296,27 +319,51 @@ void ZimDumper::listArticleT(const zim::Article& article, bool extra)
   }
   std::cout << std::endl;
 }
+void write_to_error_directoy(const std::string& base, const std::string relpath, const char *content, size_t size)
+{
+    createdir(ERRORSDIR, base);
+    std::string url = relpath;
+
+    std::string::size_type p;
+    while ((p = url.find('/')) != std::string::npos)
+        url.replace(p, 1, "%2f");
+
+    std::ofstream stream(base + ERRORSDIR + url);
+
+    stream.write(content, size);
+
+    if (stream.fail() || stream.bad())
+    {
+        std::cerr << "Error writing file to errors dir. " << (base + ERRORSDIR + url) << std::endl;
+    }
+}
 
-inline void write_to_file(const std::string& path, const char* data, ssize_t size) {
+inline void write_to_file(const std::string &base, const std::string& path, const char* data, ssize_t size) {
+    std::string fullpath = base + path;
 #ifdef _WIN32
     auto needed_size = MultiByteToWideChar(CP_UTF8, 0, path.data(), path.size(), NULL, 0);
```

Comment:
Shouldn't we use `fullpath` also on windows ?

Reply:
Yes